File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/twitter/TwitterActivity.java
package com.jdroid.android.sample.ui.twitter;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class TwitterActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return TwitterFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/twitter/TwitterActivity.kt
package com.jdroid.android.sample.ui.twitter

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class TwitterActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return TwitterFragment::class.java
    }
}